core/pom.xml

```diff
@@ -626,6 +621,12 @@
         <script.extension>.sh</script.extension>
       </properties>
     </profile>
+    <profile>
+      <id>jjwt</id>
+      <properties>
+        <jjwt.deps.scope>compile</jjwt.deps.scope>
```

Comment:
We need this both here and `assembly` in order to include the dependencies properly.